src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useContext } from 'react';
+import { StateContext } from '../../state/TodosContext';
+import { TodoItem } from '../TodoItem';
+import { Status } from '../../types';
+
+export const TodoList: React.FC = () => {
+  const { todos, filter } = useContext(StateContext);
+
+  return (
+    <ul className="todo-list" data-cy="todosList">
+      {
+        todos
+          .filter((todo) => {
+            switch (filter) {
```

Comment:
Instead of using the switch case inside the filter function, consider creating an external function to make the code more clear and reusable.